Tests for CASSANDRA-9961

stef1927 opened the pull request:
I created two tests for CASSANDRA-9961, to describe materialized views in cqlsh.

stef1927 commented:
Actually I converted one of the two tests to cover CASSANDRA-10348 as well.

mambocab commented on cqlsh_tests/cqlsh_tests.py:
```diff
@@ -1267,6 +1318,54 @@ def test_describe_round_trip(self):
+    def test_materialized_view(self):
+        """
+        Test operations on a materialized view: create, describe, select from, drop, create using describe output.
+        @jira_ticket CASSANDRA-9961 and CASSANDRA-10348
```
Will the doxygen tooling handle this, or does this need to be formatted some other way? /cc @kishkaru

ptnapoleon replied:
Doxygen will be able to handle this.

mambocab commented:
These are great, thank you. Once @kishkaru drops some Doxygen knowledge on us, I'll be +1.

mambocab commented:
Never mind, Philip says it'll work. LGTM, merge away.

stef1927 commented:
I've added one more test for CASSANDRA-10328 as well as more test cases for CASSANDRA-9961, which were requested during code review. Both CASSANDRA-10348 and CASSANDRA-10328 will be closed as duplicates of 9961. 

Please review the latest commit as well, thanks!

mambocab commented on cqlsh_tests/cqlsh_tests.py:
```diff
@@ -692,6 +692,38 @@ def test_describe_on_non_reserved_keywords(self):
+        self.execute(cql="USE test; DESCRIBE users_by_state", expected_output=self.get_users_by_state_mv_output())
+
+        # test quotes
+        self.execute(cql="USE test; DESCRIBE MATERIALIZED VIEW \"users_by_state\"", expected_output=self.get_users_by_state_mv_output())
```
You can avoid escaping double-quotes by using `'` as the delimiter for these strings. In Python `'` and `"` are identical.

mambocab commented:
Both new commits look good to me; if you like you can clean up one of the CQL strings as I mentioned in my comment. +1, thanks!

stef1927 commented:
I cleaned up the quotes and squashed, thanks. You can merge now or wait for CASSANDRA-9961 to be committed (it should happen soon) and merge without the require tags.

mambocab commented:
Sorry for the delayed review. The most recent commit looks great, thank you!

stef1927 commented:
Un-required and merged, thank you.
